src/main/java/org/mvel2/execution/ExecutionArrayList.java

```diff
@@ -122,7 +136,47 @@ public String join() {
 
     public String join(String separator) {
         return this.stream()
-                .map( Object::toString )
-                .collect( Collectors.joining( separator ) );
+                .map(Object::toString)
+                .collect(Collectors.joining(separator));
+    }
+
+    public void sort() {
+        this.sort(true);
+    }
+
+    public void sort(boolean asc) {
+        List listSort = (List) super.clone();
+        if (validateClazzInArrayIsOnlyString(listSort)) {
+            listSort = (List) listSort.stream().sorted().collect(Collectors.toList());
+            if (!asc) {
+                Collections.reverse(listSort);
+            }
+        } else {
+            Collections.sort(listSort, numericComp(asc));
+        }
+        listSort.forEach(e -> remove(e));
+        addAll(listSort);
+    }
+
+    public void toReversed() {
+        List listRev = (List) super.clone();
+        Collections.reverse(listRev);
+        listRev.forEach(e -> remove(e));
+        addAll(listRev);
+    }
+
+    private static Comparator numericComp(boolean asc) {
+        return new Comparator() {
```

Comment:
Make 2 constants (not static methods) and use them.

Reply:
Ok